Libs/RaptureCore/src/main/java/rapture/kernel/BlobApiImpl.java

```diff
@@ -125,14 +125,14 @@ public void deleteBlobRepo(CallingContext context, String blobRepoUri) {
                     }
                 }
             }
-            SearchPublisher.publishDeleteMessage(context, Kernel.getRepoCacheManager().getBlobConfig(uri.getAuthority()), uri);
         } catch (RaptureException e) {
             log.info("Unable to delete children; repo definitian may be invalid");
         }
 
         // delete parent directory
         BlobRepoConfigStorage.deleteByAddress(uri, context.getUser(), "Remove blob repo");
         removeRepoFromCache(uri.getAuthority());
+        SearchPublisher.publishDropMessage(context, uri.toString());
```

Comment:
this still wont work.  on line 133 and 134 above u delete the BlobRepoConfig.  then when u publish the search message using the SearchPublisher, u try to get the same BlobRepoConfig you just deleted.  please test this before submitting PR